docs/content/latest/architecture/transactions/distributed-txns.md

```diff
@@ -96,32 +95,36 @@ the one-byte prefix that puts these records before all regular records in RocksD
+    For details about concurrency control, see [Explicit locking](../explicit-locking).
 
-#### 3. Provisional record keys indexed by transaction ID** ("reverse index")
+#### 3. Provisional record keys indexed by transaction ID ("reverse index")
 
 ```
 TxnId, HybridTime -> primary provisional record key
 ```
 
-  This mapping allows us to find all RocksDB records belonging to a particular transaction.  This is
-  being used when cleaning up committed or aborted transactions. Note that because multiple RocksDB
-  key-value pairs belonging to primary provisional records can we written for the same transaction
-  with the same hybrid time, we need to use an increasing counter (which we call a *write id*) at
-  the end of the encoded representation of hybrid time in order to obtain unique RocksDB keys for
-  this reverse index. This write ID is shown as `.0`, `.1`, etc. in `T130.0`, `T130.1` in the figure
-  above.
+  This mapping allows us to find all provisional RocksDB records belonging to a particular
+  transaction. This is used when cleaning up committed or aborted transactions. Note that
+  because multiple RocksDB key-value pairs belonging to primary provisional records can we written
+  for the same transaction with the same hybrid time, we need to use an increasing counter (which we
+  call a *write ID*) at the end of the encoded representation of hybrid time in order to obtain
+  unique RocksDB keys for this reverse index. This write ID is shown as `.0`, `.1`, etc. in
+  `T130.0`, `T130.1` in the figure above.
 
 ## Transaction status tracking
 
 Atomicity (the "A" in "ACID") means that either all values written by a transaction are visible, or
 none are visible at all. YugabyteDB already provides atomicity of single-shard updates by
 replicating them via Raft and applying them as one write batch to the underlying RocksDB / DocDB
-storage engine. The same approach could be reused to make *transaction status* changes atomic.  The status of transactions is tracked in a "transaction status" table. This table, under the covers, is just another elastic/sharded table in the system. The transaction id (a globally unique id) serves as the key in the table, and updates to a transaction's status are simple single-shard ACID operations. This allows us to atomically make all values written as part of that transaction visible by setting the status to "committed" in that transaction's status record in the table.
-
-A transaction status record contains the following fields for a particular transaction id:
+storage engine. The same approach could be reused to make *transaction status* changes atomic.
+The status of transactions is tracked in a "transaction status" table. This table, under the covers,
+is just another sharded table in the system. The transaction ID (a globally unique ID)
```

Comment:
in the system, although it does not use RocksDB and instead stores all its data in memory, backed by the Raft WAL.